ember-flight-icons/package.json

```diff
@@ -54,7 +54,10 @@
     "test": "npm-run-all lint test:*",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each",
-    "test:ember:percy": "percy exec ember test"
+    "test:ember:percy": "percy exec ember test",
```

Comment:
(just an aside) how do we feel about trailing comma in prettier? i kinda like it cause makes diffs easier to read, but no strong opinion. i don't even think we're prettier-ing the overall repo in CI beyond `ember-flight-icons` but don't quote me on that

Reply:
don't think you can add extra commas in JSON files (I have to double check but I am pretty sure)